.travis.yml

```
@@ -10,4 +16,4 @@ script:
   - cd neutralino.js
   - npm install
   - npm test
-  
\ No newline at end of file
+  - docker run test
```

Same thing here, change the `test` tag to something more meaningful. Also, just for good practice:
```suggestion
  - docker run -d neutralinojs
  - docker ps -a
```

---

I don't think running it in detached mode is a good idea here because it may hide a failure exit status -- actually it will hide any logging the docker container will create. It would make sense if the container ran in the background, but here we want to test the build through the container so we'd like to see its output.

What do you think?

---

That's a fair point. Remove the `-d` but keep the `ps -a` right?

---

What value will keeping the `docker ps -a` bring if we see the full output of the container (from not running it in detached mode)?

I don't mind adding it, just wondering!

---

It just gives a bit more insight on the container like its UUID and the port/status of the container which the output from `docker run` doesn't have. TBH it's not that important, so do as you like, but I just like having it